app/templates/404.html

```diff
@@ -1,9 +1,9 @@
 {% extends "layout.html" %}
 {% block body %}
 <div class="container">
-    <div class="row text-center">
-        <h1 class="not-found">404</h1>
-        <h1>Page Not Found</h1>
-    </div>
+	<div class="row text-center">
```

Comment:
It was not needed, already indented.

Reply:
Yes @saarthakchats it was already indented.